modules/openapi-generator/src/main/resources/Java/libraries/native/generatedAnnotation.mustache

```diff
@@ -1 +1 @@
-@{{javaxPackage}}.annotation.processing.Generated(value = "{{generatorClass}}"{{^hideGenerationTimestamp}}, date = "{{generatedDate}}"{{/hideGenerationTimestamp}})
\ No newline at end of file
+@{{javaxPackage}}.annotation.Generated(value = "{{generatorClass}}"{{^hideGenerationTimestamp}}, date = "{{generatedDate}}"{{/hideGenerationTimestamp}})
```

Comment:
@s-jepsen @wing328 What is the intention behind this change?

Reply:
recently there was a new option `useJakartaEe` added to all java generators.

This PR aims to fix compilation issues when useJakartaEe is set to true for Java `native` client.

Is this causing issues in your use cases? Can you please share the details?

Reply:
I was curious as it will require adding the `javax.annotation-api` as a dependency for java >= 9.